index.js

```diff
@@ -223,11 +223,12 @@
           var fn = this.__outsideClickHandler;
           if (typeof document !== 'undefined') {
             var events = this.props.eventTypes || DEFAULT_EVENTS;
+            var passive = !this.props.preventDefault;
```

Comment:
This does not feel right. Passive events and preventing browser defaults are different things, even if they work toghter, so entangling them like this is not a good idea.